en/user-guide/digitizing/dataentry.rst

```diff
@@ -31,7 +29,41 @@ On the left side bottom, **QGEP Data Entry** window appears:
 
 Click **Start Data entry** to enter the edit mode. 
 
+Choose **Wastewater Structure** or **Reach**. Start adding new feature in vw_qgep_wastewater_structure or vw_qgep_reach.
+
+There is no need to select the correct layer in the layers-window, you do not need to toggle editing or to choose the add feature tool.
+
+When adding **new wastewater structures**, the wizard has no additional functionality. To **Reuse the last entered attribute values**, it is necessary (and recommended) to enable this option in :menuselection:`Settings --> Options --> Digitizing`.
+
+When adding **new reaches**, the wizard has some additional functionality:
+
+* Snapping on wastewater nodes and other reaches is on, even if QGIS **Enable Snapping** is off.
+* Snapping on nodes is always preferred to snapping on reaches.
+* When the new reach snaps, then the obj_id_links are automatically entered in the fk_fields of the reach points.
+* The **Reuse the last entered attribute values** - option of QGIS does not work. But the standard-fields on the general-tab (and only those fields) do reuse the last entered attribute value.
+
+If you use another tool (e.g. the **Identify Features** tool) and then want to continue digitizing with the wizard, you can not select the wizard again. You have to klick **Stop Data Entry** and then **Start Data Entry** and you can continue.
```

Comment:
Is there an issue for that?

Reply:
I think it's not the standard behaviour for tools - thats why I described it in detail. If you know about it, its easy and quick to "double klick" the start/stop data entry button and then continue with the wizard.
If you can easily change, that a klick on the wizard, when the data entry window is already open, it  continues with data entry of the predefined layer, would be great/better (I will change the docu suddenly, thats not a problem ;-)  ).

Reply:
That's why I asked if this has already been reported as an issue.
Because it would be nicer to "fix" it than to document the workaround, so I wondered if there is an issue reported so we don't forget about it :)